components/lib/crash/metrics.yaml

```diff
@@ -18,11 +18,14 @@ crash_metrics:
     labels:
       - uncaught_exception
       - caught_exception
-      - native_code_crash
+      - fatal_native_code_crash
```

Comment:
@mdboom @fbertsch is Glean allowed to rename labels?

Reply:
It's allowed -- they are transmitted and stored as strings.

However, I would recommend documenting in the description the purpose of the one being removed *if* we ever expect analysts to look at the old data containing that label.